src/mame/seibu/raiden2.cpp

```diff
@@ -759,7 +759,7 @@ static INPUT_PORTS_START( raiden2 )
 	PORT_DIPSETTING(      0x0006, DEF_STR( 1C_2C ) )
 	PORT_DIPSETTING(      0x0005, DEF_STR( 1C_3C ) )
 	PORT_DIPSETTING(      0x0003, DEF_STR( 1C_4C ) )
-	PORT_DIPSETTING(      0x0000, DEF_STR( Free_Play ) )
+	PORT_DIPSETTING(      0x0000, "Free Play (if Coin B too)" )
```

Comment:
What effect does this have if the other coin *isn’t* set to free play?  Could you use `PORT_CONDITION` to make it show the correct effect when only one is set this way and the other isn’t?

Reply:
We could use PORT_CONDITION but then it may be less obvious that there is a Free Play option available.  If Coin A is set to Free Play and Coin B is not, Coin A = 1Coin/1Credit.  If Coin B is set to Free Play and Coin A is not, Coin B = 1Coin/2Credits.  Suggest renaming Coin A to read "Free Play (if Coin B too) or 1C/1C" and Coin B to read "Free Play (if Coin A too) or 1C/2C" in a similar manner to Shinobi.